src/components/Page404/Page404.tsx

```diff
@@ -0,0 +1,39 @@
+import { useHistory } from 'react-router-dom';
+import { BUY_SELL_URL } from '@/constants';
+import { Localize } from '@deriv-com/translations';
+import { Button, Text, useDevice } from '@deriv-com/ui';
+import Icon404 from '../../assets/404.png';
+import './Page404.scss';
+
+const Page404 = () => {
+    const { isDesktop } = useDevice();
+    const history = useHistory();
+    return (
+        <div className='page-404'>
+            <div>
+                <img alt='Page not found' src={Icon404} />
+            </div>
+            <div className='page-404__text'>
+                <Text size={isDesktop ? '2xl' : '2xl'} weight='bold'>
```

Comment:
wait is this supposed to be a different sizing or is it just 2xl here

Reply:
oh.. yeah. its supposed to be different 👍